src/Card/Card.levers.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Styles for the card.
+ */
+// export const cardVariants = ["solid", "outline", "ghost", "unset"];
+
+export const cardAlignOptions = ["left", "center"];
```

Comment:
How much work is it to add the right aligned option as well? It just seems like it is missing.

Reply:
Not much 😄  The option has been added!

Reply:
ty